setup.cfg

```diff
@@ -77,11 +78,9 @@ warn_unused_ignores = True
 ignore_errors = True
 warn_unused_ignores = True
 
-[mypy-pyro.optm.*]
-warn_unused_ignores = True
-
-[mypy-pyro.poutine.*]
+[mypy-pyro.optim.*]
 ignore_errors = True
+warn_unused_ignores = True
```

Comment:
Can we add `warn_unused_ignores` to the top level `[mypy]` config?  I've found that some `# type: ignore`s mask real errors on the same line, so I've started using `warn_unused_ignores` globally, but maybe this practice is too pedantic for a user base with diverse mypy versions.